test/integration_test/migration_dr_actions_test.go

```diff
@@ -1579,9 +1670,35 @@ func testDRActionMetroFailback(t *testing.T) {
 	Dash.VerifyFatal(t, len(destDeployments.Items), 1, fmt.Sprintf("Expected 1 deployment in destination in %s namespace", mysqlNamespace))
 	Dash.VerifyFatal(t, *destDeployments.Items[0].Spec.Replicas, 0, fmt.Sprintf("Expected 0 replica in destination in deployment in %s namespace", mysqlNamespace))
 
-	// Verify the elasticsearch application is running on the source cluster.
+	// Validate if the clusterdomain is INACTIVE on DESTINATION cluster.
+	clusterDomainStatus, err = storkops.Instance().GetClusterDomainsStatus(destClusterDomain)
```

Comment:
Do you have to run the GetClusterDomainsStatus multiple times. We can do the validation from running the command once and getting all the domain's statuses? 

if it is not possible, I am ok with this implementation.

Reply:
Made the changes. PTAL.